### ISSUE TITLE
Redesign Add Graph Button

### ISSUE THREAD

**Issue by IshaGupta18** [publiclab/simple-data-grapher#69]
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### Problem
The current `Add Graph` button is a simple bootstrap primary button.
### 📋Solution

`<button type="button" class="btn btn-primary" id=' + this.addGraphButtonId + '> Add Graph</button>`

Change the above line in 

https://github.com/publiclab/simple-data-grapher/blob/fa222a480c402d15d260f70cb42c23694577e603/src/View.js#L347

By adding some styling, probably a `fa fa-icon` or some other color!

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update**

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

**Comment by VarsPhilos** [comment]
first-timer here, I would like to fix this issue.

**Comment by IshaGupta18** [comment]
Go right ahead! Do let us know if you face any issues at all!

**Comment by sreekanth-manda** [comment]
@VarsPhilos are you still doing this? if not I would like to work.

**Comment by IshaGupta18** [comment]
I think he is, let's wait for him to respond!

**Comment by bradingtons** [comment]
Hi first-timer here as well, if he is stuck i'm willing to help otherwise i can give this issue a try thanks.

**Comment by amnoweb** [comment]
Do you still need help with this?

**Comment by sisohs** [comment]
Is help still required?

**Comment by IshaGupta18** [comment]
Yes, would appreciate your help! Give it a shot!

**Comment by laurenelizamax** [comment]
Hi, is this still open?

**Comment by amreshk005** [comment]
@IshaGupta18 is it still open if yes can I can take it?

**Comment by sivietz** [comment]
Hello, is this issue still open?

**Comment by IshaGupta18** [comment]
Feel free to brainstorm!

**Comment by HarryKayNeezy** [comment]
hey is this issue still open? can i work on this?

**Comment by IshaGupta18** [comment]
Feel free to work on it?

**Comment by milbeeen** [comment]
Hello @IshaGupta18, is this issue still open? May I work for this one?

**Comment by debck** [comment]
Yes @milbeeen , there is no active PR for this. So you can start working.
Thanks

**Comment by itsgurmannatsohal** [comment]
Is this issue open? Can I work on it?
